ip_country/src/country_finder.rs

```diff
@@ -225,77 +191,84 @@ mod tests {
     }
 
     #[test]
-    fn real_test_ipv4_with_sk_isp() {
-        let timestart = SystemTime::now();
+    fn real_test_ipv4_with_sk_ip() {
+        let _ = CountryCodeFinder::find_country(
+            &COUNTRY_CODE_FINDER,
+            IpAddr::from_str("213.81.185.100").unwrap(), // dig www.zoznam.sk A
+        )
+        .unwrap();
```

Comment:
this is because we want to isolate time for selecting from already constructed array. so we want to avoid measure duration of construction of vector of `CountryBlocks`